src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,204 @@
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running containers as `root` is not allowed in Red Hat OpenShift.
+Currently, some components of the Okteto Control Plane require `root` privileges (this is something we will fix soon!).
+To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+These commands grant the `anyuid` Security Context Constraint (SCC) to the specified service accounts in the `okteto` namespace, allowing them to run containers as `root`.
+
+### Grant access to the "privileged" SCC for the Okteto Build service account
+
+By default, running privileged containers is not allowed in Red Hat OpenShift.
+This restriction affects the [Okteto Build](core/build-service.mdx) service.
+To grant privileged access to the Okteto Build service account, run:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+This command grants the `privileged` SCC to the `okteto-buildkit` service account in the `okteto` namespace.
+
+### Grant access to the host to the Okteto Daemon service account
+
+By default, access to the host is restricted by default in Red Hat OpenShift.
+This restriction impacts the he [Okteto Daemon](self-hosted/helm-configuration.mdx#daemonset) service.
+To grant the necessary permissionsto the Okteto Daemon, run:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+This command grants the `hostaccess` SCC to the `okteto` service account in the `okteto` namespace.
+
+### Copy Red Hat OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `*.apps.<openshift-base-domain>`.
+We recommend reusing it when installing Okteto to [expose secure and unique endpoints](self-hosted/helm-configuration.mdx#subdomain) for your development environments.
+
+Copy the certificate to the `okteto` namespace by running:
+
+```yaml
+kubectl get secret <openshift-cluster-name>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+:::info
+Note: The secret name `<openshift-cluster-name>-primary-cert-bundle-secret` is commonly used in OpenShift 4.x. Replace it with the actual secret name if it's different in your cluster.
+:::
+
+### Retrieve the Red Hat OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Red Hat Openshift Ingress Controller.
+Obtain the IP address of your Red Hat OpenShift Ingress Controller by running:
+
+```bash
+kubectl get svc router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
```

Comment:
Same prospect had a problem here. Their HAProxy route was not exposed externally and they had to add another IP internal in the Okteto config. I don't recall how they got it, I think @irespaldiza have more information here from the call we had with them

Reply:
The prospect Mario referred to was using OpenShift in an air-gapped environment. I’m unsure if we should address that scenario in a general guide.

Reply:
I would love to have the guide covering scenario without external info :)

Reply:
@mnevadom that's not possible. Specially in OpenShift there are many different configurations. That's why we don't expect this guide to be followed 100 self-service, and ask users to contact us, so we can guide them on the process